.gitlab/ci/.gitlab-ci.trigger-contribution-build.yml

```diff
@@ -0,0 +1,20 @@
+.trigger-contribution-build-role:
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(schedule|trigger)$/ && $TRIGGER_CONTRIBUTION_BUILD == "true"'
+
+stages:
+  - trigger-contribution-build
+
+trigger-contribution-build:
+  stage: trigger-contribution-build
+  variables:
+    WORKFLOW: 'Trigger contribution build'
+    GITHUB_TOKEN: $GITHUB_TOKEN
+    GITLAB_API_TOKEN: $GITLAB_API_TOKEN
+    TRIGGER_CONTRIBUTION: $TRIGGER_CONTRIBUTION
+  extends:
+    - .trigger-contribution-build-role
+    - .default-job-settings
```

Comment:
Are all the .default-job-settings needed?

Reply:
Not sure, but It is needed for the logs mechanism to work.